kge/config-default.yaml

```diff
@@ -312,6 +312,13 @@ negative_sampling:
   # 'triple' (e.g., for TransE or RotatE in the current implementation).
   implementation: triple
 
+  # Whether to compute the backward pass after scoring against each slot (S, P, O)
+  # or once after scoring against all three slots.
+  # Does not work with reciprocal relations.
```

Comment:
Move to "false".

Reply:
Also say that if false, positive triples are scored only once (else once per slot).